modules/developer-cli-odo-converting-an-s2i-component-into-a-devfile-component.adoc

```diff
@@ -0,0 +1,37 @@
+[id="converting-an-s2i-component-into-a-devfile-component_{context}"]
```

Comment:
This is missing the module metadata, so it can be difficult to know which assemblies this file is included in.